Packs/APIVoid/Integrations/APIVoid/README.md

```diff
@@ -0,0 +1,816 @@
+APIVoid wraps up a number of services such as ipvoid & urlvoid
+This integration was integrated and tested with version 01 of APIVoid
+
+## Configure APIVoid on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for APIVoid.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | API KEY |  | True |
```

Comment:
how do you generate an API KEY? we usually add this info (example) in: https://github.com/demisto/content/blob/master/Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2_description.md